src/psyclone/tests/domain/common/algorithm/alg_psyir_test.py

```diff
@@ -499,7 +530,9 @@ def test_aic_lowertolanguagelevel_multi_invoke():
     '''Check that the lower_to_language_level method works as expected
     when it has multiple invoke's. Also purposely add existing
     algorithm names that clash with root names to make sure generated
-    names are correct.
+    names are correct. Also check that the lower_to_language_level
+    method removes the invoke symbol from the appropriate symbol table
+    when the second invoke is lowered.
```

Comment:
Strictly speaking, I don't think this test checks this - it lowers everything and then checks that the `invoke` symbol has gone. Is it possible to just lower one of the invoke calls and check that the symbol is not deleted?

Reply:
Yes, good point. The test now lowers one invoke at a time and tests as it goes along.